### PR TITLE
Feature/fairsharing_tests

### PR DESCRIPTION
**Pull request by PalomoIFCA**
The goal is to add all the test that require fairsharing

### CODE REVIEW

File: plugins/epos/config.ini [hunk 1]
```diff
@@ -138,3 +138,14 @@ data_access_manual = ['https://epos-eu.github.io/EPOS-DCAT-AP/v3/#properties-for
...
+metadata_standard = ['DCAT']
+
+
+[fairsharing]
+# username and password
+fairsharing_username = ['']
+
+fairsharing_password = ['']
+
+fairsharing_path = ['static/fairsharing_metadata_standards140224.json']
```
Comment (orviz) [inline plugins/epos/config.ini:151]: no need of prefixing the properties with `farisharing_`, they are already under `[fairsharing]` category

File: plugins/epos/plugin.py [hunk 2]
```diff
@@ -91,6 +91,19 @@ def __init__(self, item_id, oai_base=None, lang="en", config=None):
...
+        )
+        self.fairsharing_username = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_username"]
+        )
+
+        self.fairsharing_password = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_password"]
+        )
+        self.fairsharing_path = ast.literal_eval(
+            self.config["fairsharing"]["fairsharing_path"]
```
Comment (orviz) [inline plugins/epos/plugin.py:105]: there should be a fallback when getting to get parameters that are optional, e.g. username and password. Have you tried if these statements fail when username and password are not defined?

File: api/utils.py [hunk 3]
```diff
@@ -752,3 +753,45 @@ def get_protocol_scheme(url):
...
+        json.dump(fairlist, user)
+        user.close()
+    return fairlist
+
+
+def check_fairsharing_abbreviation(fairlist, abreviation):
+    for standard in fairlist["data"]:
+        if abreviation == standard["attributes"]["abbreviation"]:
+            return (100, "Your metadata standard appears in Fairsharing")
+    return (0, "Your metadata standard has not been found in Fairsharing")
```
Comment (orviz) [inline api/utils.py:797]: this code should be moved to the RDA indicator function. Then, I would use a more concrete message for the specific indicator, e.g. "Metadata standard in use complies with a community standard according to FAIRsharing.org"

**Review by orviz — CHANGES_REQUESTED** [review]

File: plugins/epos/config.ini (hunk 1 above)
Comment (PalomoIFCA) [reply plugins/epos/config.ini:151]: It is that way to make  the variable name is consistent between the two files. In the case that there is a need for other passwords I would prefer this name

File: plugins/epos/plugin.py (hunk 2 above)
Comment (PalomoIFCA) [reply plugins/epos/plugin.py:105]: If they are defined as an empty list the tool uses the premade list file, but the program fails when the path does not take to any file, probably should give you 0 points but change the msg

Comment (PalomoIFCA) [reply plugins/epos/plugin.py]: Fixed in commit Improved test r1_3_01m

File: api/utils.py (hunk 3 above)
Comment (PalomoIFCA) [reply api/utils.py:797]: Fixed in commit @PalomoIFCA
[Bug fix and simplified code by not using func](https://github.com/EOSC-synergy/FAIR_eva/pull/142/commits/166682559e247ce94a49679c5f3cd56da736b802)

File: plugins/epos/config.ini (hunk 1 above)
Comment (PalomoIFCA) [reply plugins/epos/config.ini]: Updated in: [Changed names to reduce redundance in config.ini](https://github.com/EOSC-synergy/FAIR_eva/pull/142/commits/6789df37a26ce0a0cc2ada25929a82bbaa6c07e9)

**Review by orviz — APPROVED** [review]

**Pull request merged**